app/components/FilterOptions.tsx

```diff
@@ -57,6 +68,7 @@ const FilterOptions = ({
             selected={option.key === activeKey}
             {...menu}
           >
+            {Image && <StyledImage model={option.user} size={20} />}
```

Comment:
I don't know why you're designing the interface like this, so you can pass an `image` prop into `FilterOptions` but it must have a component with a `model` prop? It doesn't really make sense… `image` should just allow any component here.

Reply:
Right. It's awful. I'm making new changes, and now I'm passing `styledAvatar` component in from `UserFilter` to save it from unnecessary re-rendering in `filteroptions`...

Reply:
I've pushed the changes. and everything is as we discussed here https://github.com/outline/outline/pull/4400#discussion_r1017361813